deploy/cr.yaml

```diff
@@ -26,6 +26,7 @@ spec:
     clusterType: async
     image: percona/percona-server:8.0.29
     imagePullPolicy: Always
+#    initImage: percona/percona-server-mysql-operator:0.3.0
```

Comment:
maybe we should use main images in main branch and change it to release tag in release branch?

Reply:
fixed